eip-5564.md

```diff
@@ -118,6 +118,11 @@ contract IERC5564Messenger is StakeManager{
   /// Besides the view tag, the metadata can be used by the senders however they like, 
   /// but the below guidelines are recommended:
   /// The first byte of the metadata MUST be the view tag.
+  /// - When sending the native token of the blockchain (cf. ETH), the metadata SHOULD be structured as follows:
+  ///     - Byte 1 MUST be the view tag, as specified above.
+  ///     - Bytes 2-5 are arbitrary bytes (e.g. 0xdeadbeef).
```

Comment:
Should we just remove this for the native token case? It feels unnecessary. Or we can make it another 4 bytes of e's

Reply:
Do you mean removing both, the view tag and the method id (since both are marked) or the whole section on the native token in general?

I'm happy to remove the method ID, however, since there's place for it without reaching 32 bytes, it may help parsing providers by not having them to implement a separate "parse_metadata()" for Eth transactions compared to ERCs.

Reply:
That's true. So now I'd say we do either four more bytes of e's, OR use the transfer selector that we use for tokens

Reply:
I also thought about the transfer selector, but actually the method id will then not be found in the transaction so it should probably be a method id that isn't used anywhere.
I'm just thinking of implementing that on the parsing side: If we use the transfer method, there could be parsing providers indicating a ERC-20 transfer if they don't have some logic that says "if the token address is 0xeee then ignore the method id". If we provide an invalid methodID this is more clear, I guess.

I'd more prefer an invalid method id, so fine with both, 0xeee.. or deafbeef, or even 0x00000000 if that works.